qa/src/test/resources/application.yml

```diff
@@ -0,0 +1,17 @@
+camunda:
```

Comment:
🔧 If we add this to the code base should/can we remove `qa/src/main/resources/application.yml`. It will be better to consolidate them at one place.